### ISSUE TITLE
No code editor button in toolbar 

### ISSUE THREAD

**Issue by Sawrz** [EnlighterJS/Plugin.WordPress#104]
Hi,
as the title states:
After updating Enlighter the button to insert code in the editor is missing. See here:

![screenshot from 2018-06-15 07-05-02](https://user-images.githubusercontent.com/9134364/41450878-73304f2a-706a-11e8-9e70-49b05fd8402e.png)

Cheers,
Sandro

**Comment by AndiDittrich** [comment]
Hi Sandro,

the button names/labels have been changed from 3.6 to 3.7 - in case you have any editor customization plugins enabled this may cause the issue. please try to disable other editing plugins and clear your browsers cache

i've tried to reproduce your issue with a clean WordPress 4.9.6 installation but everything works as expected.

best regards, Andi

**Comment by Sawrz** [comment]
Hi Andi,
I found the Problem. The bad guys are Thrive Leads and Thrive Quiz Builder. In case somebody uses them too and have the same issues. 

I will submit a ticket to them. Social Warfare Buttons vanished too (I don't use them so I didn't notice they disappearing...).

Thank you for your quick reply!

**Comment by Sawrz** [comment]
Hi Andi, 
I have to add that the new changes cause more trouble for me than I initially thought. Another Plugin (Analytify Pro) causes the code button to disappear too. It seems like the compatibility breaks with the recent changes.

Is ther ea possibility for me to rollback? I don't want to deactivate my conversion plugins each time by hand. =(
Maybe you have an idea for a workaround, so I can enjoy both worlds?

Best,
Sandro

**Comment by AndiDittrich** [comment]
Hi Sandro,

i didn't know whats going on because om currently unable to reproduce the issue. are some of the plugins you've used freely available ? i've take a deep dive into the Enlighter code to identify the issue.

can you see any javascript errors in your browsers console when the Enlighter buttons disappear ?

btw. you can still fetch all previous release from [GitHub](https://github.com/EnlighterJS/Plugin.WordPress/releases) or from WordPress.org/plugins/enlighter

best regards, Andi

**Comment by AndiDittrich** [comment]
Hi Sandro,

the errors are not Enlighter.TinyMCE related..thats very strange :( but something seems to be broken on your site.

to debug the Enlighter related issue i need access to the Editor (user account with minimal privileges: contributor role) - that allows me to create posts without publishing them.

br, Andi

**Comment by Sawrz** [comment]
Hi Andi,
I prepared a contributor account for you. However, I need an email address or another way to send you the credentials =)

Best,
Sandro

**Comment by AndiDittrich** [comment]
`opensource _AT andidittrich D0TT de`

**Comment by Sawrz** [comment]
Hi, it's me again. Just let you know the guys from Thrive Themes released a hotfix. The issue doesn't appear for me anymore. Analytify Pro was a false true from my side. Sorry for that.

However, I think there was an issue with another Thrive plugin, which I removed. If there is a similar problem, I would check that first. Nevertheless, I informed the Thrive guys that the other plugin caused also a similar issue. Should be fixed then too.
